scripts/version.sh

```diff
@@ -31,9 +31,9 @@ REVISION=$(git rev-parse HEAD)$(if ! git diff --no-ext-diff --quiet --exit-code;
 PLATFORM=${GOOS}-${GOARCH}
 RELEASE=${PROG}.${PLATFORM}
 # hardcode versions unless set specifically
-KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.20.12}
-KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.20.12-rke2r1-build20211028}
-ETCD_VERSION=${ETCD_VERSION:-v3.4.13-k3s1}
+KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.20.13}
+KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.20.13-rke2r1-build20211119}
+ETCD_VERSION=${ETCD_VERSION:-v3.4.13-k3s3}
 PAUSE_VERSION=${PAUSE_VERSION:-3.2}
 CCM_VERSION=${CCM_VERSION:-v0.0.3-build20211118}
 IMAGE_BUILD_VERSION=${IMAGE_BUILD_VERSION:-build20210223}
```

Comment:
I think this needs to be bumped too, same as in 1.21
```suggestion
IMAGE_BUILD_VERSION=${IMAGE_BUILD_VERSION:-build20211118}
```

Reply:
Resolved.